src/Rules/Classes/PluginManagerInspectionRule.php

```diff
@@ -78,9 +78,9 @@ public function processNode(Node $node, Scope $scope): array
             $errors[] = RuleErrorBuilder::message(
                 'Plugin managers should call alterInfo to allow plugin definitions to be altered.'
             )
-                ->identifier('plugin.manager.alterInfoMissing')
                 ->tip('For example, to invoke hook_mymodule_data_alter() call alterInfo with "mymodule_data".')
                 ->line($node->getStartLine())
+                ->identifier('pluginManagerInspection.alterInfoMissing')
                 ->build();
```

Comment:
I didn't review the commit log, but I'm guessing something about the format of the error identifier made PHPStan mad?

Reply:
Nope, this is a rare occasion where there was actually an identifier and I completely overlooked it.

Not too sure what to do now...

Do we want double-full-stop identifiers? 

If we do I'll have to change the other identifiers in this class to the same prefix.

Help! :)

Reply:
I'm fine taking your changes, I was just curious